tests/framework/di/InstanceTest.php

```diff
@@ -46,6 +46,26 @@ public function testEnsure()
         $this->assertTrue(Instance::ensure(['class' => 'yii\db\Connection', 'dsn' => 'test'], 'yii\db\Connection', $container) instanceof Connection);
     }
 
+    /**
+     * ensure an InvalidConfigException is thrown when a component does not exist.
+     */
+    public function testEnsureNonExistingComponent()
```

Comment:
do you have a suggestion for a better name? What you mean would be more like the message on an assertation, no?